scrapers/base.py

```diff
@@ -43,3 +44,24 @@ def get_html_content(self, url):
             print "Url is not reachable. Error logged on: {}".format(
                 strftime("%Y-%m-%d %H:%M:%S", gmtime()))
             log.info(str(err))
+
+    def local_store(self, data):
+        """Writes content from the website to file.
+        Usage::
+             the file name to write the data
+        :param_train_data: the filename
+        :rtype: outputs data to file.
+        """
+        file_name = "news"
+        if file_name:
+            with open(os.path.dirname(__file__) +"/data/"+"news"+".json", "r+") as output_file:
```

Comment:
`"/data/"+"news"+".json"` looks unnecessary. Let's instead have it as `"/data/news.json"`. Unless there's a specific reason not to?